django_freeradius/base/models.py

```diff
@@ -511,3 +531,124 @@ class Meta:
 
     def __str__(self):
         return str(self.username)
+
+
+class AbstractRadiusBatch(TimeStampedEditableModel):
+    name = models.CharField(verbose_name=_('name'),
+                            max_length=128,
+                            help_text=_('A unique batch name'),
+                            db_index=True)
+    strategy = models.CharField(_('strategy'),
+                                max_length=16,
+                                choices=STRATEGIES,
+                                db_index=True,
+                                help_text=_('Import users from a CSV or generate using a prefix'))
+    users = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                   blank=True,
+                                   related_name='radius_batch',
+                                   help_text=_('List of users uploaded in this batch'))
+    csvfile = models.FileField(null=True,
+                               blank=True,
+                               verbose_name='CSV',
+                               help_text=_('The csv file containing the user details to be uploaded'))
+    prefix = models.CharField(_('prefix'),
+                              null=True,
+                              blank=True,
+                              max_length=20,
+                              help_text=_('Usernames generated will be of the format prefix_number'))
```

Comment:
`Usernames generated will be of the format "<prefix><number>"` in the truth there's no `_`

Reply:
Unfortunately if I put `<prefix><number>` in the help, the template assumes it like a html tag and doesn't print it.

Reply:
Using the `[prefix][number]` notation